prov/efa/src/efa_domain.h

```diff
@@ -26,11 +26,28 @@ struct efa_domain {
 	size_t			mtu_size;
 	size_t			addrlen;
 	bool 			mr_local;
-	uint64_t		rdm_mode;
-	size_t			rdm_cq_size;
 	struct dlist_entry	list_entry; /* linked to g_efa_domain_list */
 	struct ofi_genlock	srx_lock; /* shared among peer providers */
+
+	/* Only valid for RDM EP type */
+	uint64_t		rdm_mode;
+	size_t			rdm_cq_size;
```

Comment:
Any significant impact observed by moving the struct members here?

Reply:
I haven't observed. I can re-arrange it if we see any. I moved here just for better readiness and the convenience of future refactoring (separate dgram domain and rdm domain)